src/modules/dashboard/components/ColonyHome/ColonyMeta/ColonyMeta.tsx

```diff
@@ -3,6 +3,8 @@ import { defineMessages, FormattedMessage } from 'react-intl';
 
 import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
 import { DomainsMapType } from '~types/index';
+import { ColonyType } from '~immutable/index';
+import { AnyColony } from '~data/index';
```

Comment:
Do we still need `AnyColony` here ?

Reply:
Yeah, as we use ColonyMeta in both cases, one where we have a "full colony" and one where we have just the colony from the db